config/node/network-environment-template

```diff
@@ -10,7 +10,7 @@ KUBERNETES_MASTER=<KUBERNETES_MASTER>
 ETCD_AUTHORITY=<KUBERNETES_MASTER>:6666
 
 # URL to access the Kubernetes apiserver
-KUBE_API_ROOT=http://<KUBERNETES_MASTER>:8080/api/v1/
+KUBE_API_ROOT=https://<KUBERNETES_MASTER>:443/api/v1/

```

Comment:
This is used for the ubuntu guide, which doesn't use HTTPs, so it should stay at 8080 for now.